src/deepEqual.js

```diff
@@ -23,8 +23,48 @@
  *
  */
 
+function isEquals(value1, value2) {
+  return typeof value1 === typeof value2 && value1 === value2;
+}
+
+function isObject(elem, elem2) {
```

Comment:
why pass elem2? is this function checking that elem is an object? or something else? 

Try to get rid of elem2.